### ISSUE TITLE
Hopefully Helpful Suggestions

### ISSUE THREAD

**Issue by shellscape** [LeaVerou/md-block#2]
Ola. Big fan of markdown and this caught my eye by way of the dreaded orange beast. Some suggestions for the project that are hopefully received as helpful:

- ESLint and/or Prettier (this will surely help contributors)
- Tests! (ava/uvu/jest/etc anything with snapshots will probably be useful)
- Minification (terser would probably suffice)

🍻

**Comment by LeaVerou** [comment]
Yeah, agreed on all three, I was just being lazy (and in my local environment ESLint is applied globally anyway, so nothing seemed to be missing).
Mind opening separate issues so we could track progress?
